src/components/ValidatedInput/ValidFieldPassword.tsx

```diff
@@ -51,17 +52,11 @@ export class ValidFieldPassword extends React.Component<ValidFieldPasswordProps,
     this.hideTooltip();
   }
 
-  static getDerivedStateFromProps(
-    props: { strengthLevelIndicator: keyof typeof strengthLevelOrder },
-    state: { levelColor: string }
-  ) {
-    if (props.strengthLevelIndicator !== state.levelColor) {
-      return {
-        levelColor: props.strengthLevelIndicator,
-      };
+  componentDidUpdate(prevProps: ValidFieldPasswordProps) {
+    const { hasPasswordStrenghtMeter } = this.props;
```

Comment:
typo here. should be `strength` not `strenght`

Reply:
done